src/OneDrive.Sdk.Authentication.Desktop/Business/AuthenticationContextWrapper.cs

```diff
@@ -54,6 +54,19 @@ public ITokenCache TokenCache
             }
         }
 
+
+        /// <summary>
+        /// Authenticates the daemon app silently"/>.
+        /// </summary>
+        /// <param name="resource">The resource to authenticate against.</param>
+        /// <param name="clientCredential">The client credential of the application.</param>
+        /// <returns>The <see cref="IAuthenticationResult"/>.</returns>
+        public async Task<IAuthenticationResult> AcquireDaemonTokenSilentAsync(string resource, ClientCredential clientCredential)
```

Comment:
Why add this when you can use `AcquireTokenSilentAsync(string resource, ClientCredential clientCredential, UserIdentifier userIdentifier)` with `UserIdentifier.Any`? It should be the same unless I'm missing something.